Vision/proposal.md

```diff
@@ -0,0 +1,62 @@
+# The Mission of the W3C
+
+The W3C leads the community in defining a World Wide Web that is inclusive and respectful of its users, 
+to empower an ethical, equitable, informed, and interconnected society.
+
+## **Introduction**
+### The World Wide Web
+
+The World Wide Web was originally conceived as a tool for sharing information. 
+It has evolved rapidly into a fundamental part of humanity, 
+sparking major social change by providing access to 
+knowledge, education, commerce and shopping, social experiences, civic functions, entertainment, and more.
+
+The Web's amazing success has also led to many unintended consequences that harm society: 
```

Comment:
```suggestion
The Web's amazing success has also led to many undesirable consequences that harm society: 
```

Reply:
Added word, but "unintended" is very important to have here.